.github/workflows/smoke-checks.yml

```diff
@@ -48,9 +48,9 @@ jobs:
 
   automated-code-review:
     name: Automated Code Review
-    runs-on: macos-12
+    runs-on: macos-14
     env:
-      XCODE_VERSION: "14.0.1"
+      XCODE_VERSION: "15.0.1"
```

Comment:
I was thinking why not 15.4, but I guess because App Store accepts all the Xcode 15 versions